cloudformation/launchdetector.yaml

```diff
@@ -161,14 +175,16 @@ Resources:
         "Fn::Sub": |
           #!/bin/bash -e
 
-          apt-get -y update
-          apt-get -y install awscli openjdk-8-jre-headless
+          #this should be installed on the base box
+          #apt-get -y update
+          #apt-get -y install awscli openjdk-8-jre-headless
 
           mkdir -p /tmp/install
           aws s3 sync s3://gnm-multimedia-rr-deployables/${Stack}/${Stage}/${App} /tmp/install
 
-          MOST_RECENT_PACKAGE=`ls -Frt /tmp/install/*.deb | grep "[^/]$" | tail -n 1`
-          dpkg -i "$MOST_RECENT_PACAKAGE"
+          declare -x MOST_RECENT_PACKAGE=`ls -Frt /tmp/install/*.deb | grep "[^/]$" | tail -n 1`
```

Comment:
how are we getting multiple artefacts in this bucket?

Reply:
RiffRaff is putting them there.  It's being built as a .deb package, which includes the build number in both the filename and metadata.  So, RR does not over-write the old one during deploy.  I'm sure that there must be a way around it, but my priority has been to get the code working and deployed first and tidy up later.